scripts/env_setup.sh

```diff
@@ -11,6 +11,25 @@ getDependencies(){
     ./get_dependencies.sh
 }
 
+#Get user bot token and store in json file
+getBotToken(){
+	# Prompt user for their bot token
+	read -p "Enter your bot token: " token
+
+	# Create json object
+	json_data=$(jq -n --arg token "$token" \
+		'{token: $token}')
+
+	# Define output file
+	output_file="bot_token.json"
```

Comment:
Let's make the name of the `output_file` to be `config.json` since we might eventually store other configuration data here.